.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 cbor-js.zip
 cbor.min.js
 sauce_connect.log
+.idea
```

Comment:
Copied fix from master to readFloat16.

https://jsfiddle.net/czpdkw3w/74/

Starting at Thu Sep 22 2016 12:05:01 GMT-0400 (Eastern Daylight Time)
readFloat16 x 3,197 ops/sec ±0.80% (21 runs sampled)
decodeFloat16 x 133,272 ops/sec ±1.29% (63 runs sampled)
decodeFloat16_2 x 146,115 ops/sec ±1.77% (62 runs sampled)
decodeFloat16_3 x 145,785 ops/sec ±1.10% (62 runs sampled)
Completed at Thu Sep 22 2016 12:05:26 GMT-0400 (Eastern Daylight Time)
Fastest: decodeFloat16_3, decodeFloat16_2
Unexpected results: 0
Not a Number results: 1374
Negative zeroes: 458
Expected results: 233580
Results checked: 458
Buffers built: 458